### ISSUE TITLE
Backcountry: Start looking into admin panel

### ISSUE THREAD

**Issue by Dianadec** [bcgov/reserve-rec-admin#4]
#### Description:

Flesh something out in admin that mirrors what API needs to do. Designs would be simple - copy whatever the model is.

#### Acceptance Criteria:
...

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by Dianadec** [comment]
Oct 16 Refinement Notes:
- Starting point for admin panel:
- allow non-tech people to access API easier (Currently only accessible in postman format). Allow admin folks to do main operations. Doesn't really need designs at this point. "what are the possible fields?" "Save"
- Other things WILL need designs, which we can't work on right now. For this, we would need to gather user needs of the business admin users. Then design, iterate, feedback, etc.
- so if we allowed website admins to look at data via the api and make changes via the api that's prob the biggest thing the admin panel is going to be used for
- Example: going to need the role equivalent of the camis field system 

TLDR: we can do a bunch of work on admin side for now without designs, according to the notes above.

**Comment by Dianadec** [comment]
@LindsayMacfarlane should we rename this to simple 'admin panel' and remove backcountry??

**Comment by LindsayMacfarlane** [comment]
@Dianadec - that sounds like a plan...and done ;)

**Comment by LindsayMacfarlane** [comment]
**Backlog refinement (January 7, 2025):** Not sure if both teams have the same definition for “admin panel”. For the reservation services team, our current “admin panel” (system) is used by BC Parks HQ staff, BC Parks regional staff, the call centre and Park Operators. All with very different user needs. Accessing report, updating data, making reservations etc. Not sure if this is helpful or not to the discussion.

**Comment by LindsayMacfarlane** [comment]
Closing. Considering this EPIC now complete. Future EPICS will be created as needed. 

FYI @kcabhar
